### PR TITLE
Update SDK and documentation.

### PR DESCRIPTION
**Pull request by andrewrmiller**
This change supports issues #3 and #4.

### CODE REVIEW

File: doc/008-debugging.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+## Logging
+
+The SDK provides a `Logger` class that you can use to write out logging information as your app runs. The Logger class has four logging methods that differ only in the severity that is attached to the log output:
+
+```
+Logger.debug(message, detail)
+Logger.info(message, detail)
+Logger.debug(message, detail)
+Logger.debug(message, detail)
```
Comment (seag11) [inline doc/008-debugging.md:17]: Should this be the following?
```
Logger.error
Logger.warn
```

Comment (andrewrmiller) [reply doc/008-debugging.md:17]: Whoops.  Looks like I didn't finish this edit.

Comment (andrewrmiller) [reply doc/008-debugging.md]: Fixed.

**Review by seag11 — APPROVED** [review]
Looks good

**Pull request merged**